src/khoj/interface/web/chat.html

```diff
@@ -128,12 +129,20 @@
         }
 
         function incrementalChat(event) {
-            // Send chat message on 'Enter'
-            if (event.key === 'Enter') {
+            if (!event.shiftKey && event.key === 'Enter') {
                 chat();
             }
         }
 
+        function autoResize() {
+            const textarea = document.getElementById('chat-input');
+            const scrollTop = textarea.scrollTop;
+            textarea.style.height = '0';
+            const scrollHeight = textarea.scrollHeight;
+            textarea.style.height = Math.min(scrollHeight, 200) + 'px';
+            textarea.scrollTop = scrollTop;
+        }
```

Comment:
Should we make the `chat-body` scroll to the bottom after resizing as well? Maybe something like below:

```suggestion
            textarea.scrollTop = scrollTop;
            document.getElementById("chat-body").scrollTop = document.getElementById("chat-body").scrollHeight;
        }
```